src/Data/filterSchema.ts

```diff
@@ -0,0 +1,52 @@
+import { pisaClient } from './pisaClient'
+import { DataClassAttributes } from './types'
+
+/**
+ * Fetches the schema for `subPath`.
+ * Filters out boolean and reference attributes, since they are currently not fully supported.
+ * It also converts enum values, if in a wrong format.
+ **/
+export async function filterSchema(subPath: string) {
```

Comment:
* [ ] FYI Missing return type. Not sure which policy we use. There are other exported functions in this PR without return type.
* [ ] FYI since we're only returning the `attributes` part the function name is misleading.